app/controllers/application_controller.rb

```diff
@@ -18,12 +20,27 @@ def sign_in(login_userinfo)
     user = User.user_from_userinfo(login_userinfo)
 
     @current_user = user
+    renew_session
     session[:userinfo] = login_userinfo
   end
 
   def sign_out
     @current_user = nil
     session.delete(:userinfo)
+    session.delete(:session_timeout_at)
+  end
+
+  def renew_session
+    session[:session_timeout_at] = Time.current + ENV.fetch("SESSION_TIMEOUT_IN_MINUTES", 15).to_i.minutes
```

Comment:
I generally try to keep ENV usage to a minimum. Do we keep this in ENV on the Phoenix app? I don't think it needs to change frequently or be configured on different environments. maybe it could be a constant on the controller or LoginGov? not sure the best place, e.g.
```
LoginGov::SESSION_TIMEOUT_IN_MINUTES # or
SessionController::SESSION_TIMEOUT_IN_MINUTES
```

Reply:
Yeah it exists on the phoenix app. I think initial requirements for the implementation on that end wanted the ability to configure it